docs/intro.rst

```diff
@@ -6,8 +6,42 @@ Introduction
 This section provides an overview of Confluent's Docker images for the Confluent Platform.  We've included an overview of image design, a quickstart guide, advanced tutorials, and developer instructions for extending the images.  
 The images are currently available on `DockerHub <https://hub.docker.com/u/confluentinc/>`_.  Alternatively, the source files for the images are `available on Github <https://github.com/confluentinc/cp-docker-images>`_ if you would prefer to extend and/or rebuild the images and upload them to your own DockerHub repository.
 
-The images are currently only available for Confluent Platform 3.0.1.
-
+The images are currently only available for Confluent Platform 3.0.1 and greater.
```

Comment:
Makes sense.  We should add an alias/tag for cp-enterprise-control-center for 3.0.1.